skyvern/forge/sdk/api/llm/api_handler_factory.py

```
@@ -934,14 +934,25 @@ async def llm_api_handler(
+                        prompt_name=prompt_name,
+                        duration=t_llm_cancelled - t_llm_request,
+                    )
+                    raise
+                else:
+                    LOG.error(
+                        "LLM request got cancelled",
+                        llm_key=llm_key,
+                        model=model_name,
+                        prompt_name=prompt_name,
+                        duration=t_llm_cancelled - t_llm_request,
+                    )
```

Ruff (0.14.8)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)